### PR TITLE
remove outdated dev stuff from the README

### LINKED ISSUE #53: README may need minor update
The example provided fails with:
```
>>> from utide import solve, reconstruct
>>> from utide.tests.test_solve import test_roundtrip
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'utide.tests'
```

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #53

### CODE REVIEW

**Comment by efiring** [comment]
The change is fine.  It looks like a newer and fussier version of flake8 is now complaining about something unrelated to this change.

**Comment by ocefpaf** [comment]
> It looks like a newer and fussier version of flake8 is now complaining about something unrelated to this change.

I'll fix that in a moment.

**Comment by efiring** [comment]
Flake8's complaint looks bogus; I don't see anything "ambiguous" about having an attribute called "I".

**Comment by ocefpaf** [comment]
> Flake8's complaint looks bogus; I don't see anything "ambiguous" about having an attribute called "I".

It is linting for variables with short names that can be ambiguous with expected "default" variable names, `I`, for example, is usually used for the imaginary number.

We should not worry about that b/c the code linting should be in the allow failure section. It is meant to be used as a guide and not a hard test. I fixed that in my latest commit.

File: .travis.yml [hunk 1]
```diff
@@ -13,8 +13,6 @@ matrix:
     env: TEST_TARGET=default
   - python: 3.6
     env: TEST_TARGET=default
-  - python: 3.5
-    env: TEST_TARGET=coding_standards
```
Comment (ocefpaf) [inline .travis.yml]: This line should not be here.

**Comment by ocefpaf** [comment]
@efiring are you OK with this CI configuration?

**Comment by efiring** [comment]
Yes, thank you.

**Comment by ocefpaf** [comment]
With the approvals I guess it is safe to self-merge this one :smile:

**Pull request merged**